arangod/Cache/Finding.h

```diff
@@ -43,6 +44,7 @@ class Finding {
  public:
   Finding();
   explicit Finding(CachedValue* v);
+  explicit Finding(CachedValue* v, Result const& r);
```

Comment:
I'd rather keep it there and avoid any implicit conversions. Otherwise we might accidentally drop custom result messages by passing a code instead. Or worse, pass a different pointer type.